openapi/nexus.json

```diff
@@ -8490,6 +8490,96 @@
         }
       }
     },
+    "/v1/system/timeseries/query": {
+      "post": {
+        "tags": [
+          "system/metrics"
+        ],
+        "summary": "Run timeseries query",
+        "description": "Queries are written in OxQL.",
```

Comment:
Do we want the summary or description to reflect that this is for fleet-scoped timeseries? Not sure if it's relevant or important yet, given that's the only scope we have today.

Reply:
Well, currently it is technically for all timeseries in the system. I'm honestly not sure whether we want to change that in the future. Generally we try not to make it too easy for operators to peek inside silos, with a few exceptions like being able to list instances on a given sled.